downloads/index.md

```diff
@@ -8,14 +8,14 @@ title:  Julia Downloads
 We provide several ways for you to run Julia:
 
 * In the terminal using the built-in Julia command line.
-* The [Juno](http://www.junolab.org) integrated development environment (IDE).
 * In the browser on [JuliaBox.com](https://www.juliabox.com) with Jupyter notebooks. No installation is required -- just point your browser there, login and start computing.
+* The [JuliaPro distribution](http://juliacomputing.com/products/juliapro.html) by Julia Computing provides a single installer for Julia, the [Juno IDE](http://junolab.org), the Gallium debugger, and a number of packages (requires registration).
```

Comment:
It might be nice to make Julia Computing a link to the Julia Computing website and similarly for Gallium to https://github.com/Keno/Gallium.jl.